File: .sdk/tm/rb/utility/make_context.rb
# ChineseHolidayCalendar SDK utility: make_context
require_relative '../core/context'
module ChineseHolidayCalendarUtilities
  MakeContext = ->(ctxmap, basectx) {
    ChineseHolidayCalendarContext.new(ctxmap, basectx)
  }
end

File: py/chineseholidaycalendar_sdk/utility/make_context.py
# ChineseHolidayCalendar SDK utility: make_context

from chineseholidaycalendar_sdk.core.context import ChineseHolidayCalendarContext


def make_context_util(ctxmap, basectx):
    return ChineseHolidayCalendarContext(ctxmap, basectx)
